config/module.config.php

```diff
@@ -31,6 +31,12 @@
                 'hydrator_dir'       => 'data/DoctrineMongoODMModule/Hydrator',
                 'hydrator_namespace' => 'DoctrineMongoODMModule\Hydrator',
 
+                'generate_persistent_collections' => Configuration::AUTOGENERATE_ALWAYS,
```

Comment:
What do you mean by, *other Zend moduels*? This is *the* Zend module for Doctrine integration. I don't think `AUTOGENERATE_NEVER` is a good default since it means your application is broken by default. That's a production default, not a developer default.